File: android/app/modules/computer/src/main/java/com.multiapk.modules.computer/ComputerFragment.java
package com.multiapk.modules.computer;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.multiapk.library.base.MFragment;


public class ComputerFragment extends MFragment {

    @SuppressLint("InflateParams")
    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        View contentView = inflater.inflate(R.layout.fragment_computer, null);
        contentView.findViewById(R.id.textComputer).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(mFragmentActivity, MyReactActivity.class));
            }
        });
        return contentView;
    }
}

File: android/app/modules/computer/src/main/java/com.multiapk.modules.computer/ComputerFragment.kt
package com.multiapk.modules.computer

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.jakewharton.rxbinding2.view.RxView

import com.multiapk.library.base.CMFragment
import org.jetbrains.anko.toast

class ComputerFragment : CMFragment() {
    @SuppressLint("InflateParams")
    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val contentView = inflater!!.inflate(R.layout.fragment_computer, null)
        RxView.clicks(contentView).subscribe {
            context?.toast("电脑模块")
            startActivity(Intent(context, MyReactActivity::class.java))
        }
        return contentView
    }
}
